tests/test_converters.py

```diff
@@ -209,6 +209,52 @@ def test_read_image_value_error(self):
                       self.train_images_path, 'int32')
 
 
+class TestAdult(object):
+    def setUp(self):
+        self.tempdir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.tempdir)
+
+    def test_fill_subparser(self):
+        parser = argparse.ArgumentParser()
+        subparsers = parser.add_subparsers()
+        subparser = subparsers.add_parser('adult')
+        adult.fill_subparser(subparser)
+        assert parser.parse_args(['adult']).func is adult.convert_adult
+
+    def test_convert(self):
+        tempdir = self.tempdir
+
+        cwd = os.getcwd()
+        os.chdir(tempdir)
+
+        assert_raises(IOError,
+                      adult.convert_adult,
+                      directory=tempdir,
+                      output_directory=tempdir)
+
+        default_downloader(
```

Comment:
Alright, let's leave it at that for now, and we'll do something later if it becomes problematic.